recipes/Time_Series/Time_Series_Getting_Started.ipynb

```diff
@@ -182,7 +182,7 @@
    "source": [
     "### Create a forecasting pipeline\n",
     "\n",
-    "Set up the forecasting pipeline with the model, setting `frequency` given our knowledge of the sample frequency. In this example we set `explode_forecasts` to `True`, which will format the output for plotting the history and prediction period. We then make a forecast on the dataset."
+    "Set up the forecasting pipeline with the model, setting `frequency` given our knowledge of the sample frequency. In this example we set `explode_forecasts` to `False`, which will format the output for plotting the history and prediction period. We then make a forecast on the dataset."
```

Comment:
@wgifford The value was recently changed from True to False but this explanatory text was not updated. Is the use of False mean what the text says? Or does the text need correction?

Reply:
Ok, will update it @wgifford 

Thanks @bjhargrave for raising it

Reply:
Updated with @wgifford comment.